src/ServiceProvisions.js

```diff
@@ -0,0 +1,173 @@
+/* @flow */
```

Comment:
I'm concerned that `src/*` is getting pretty long; is there a sensible folder for this?